### PR TITLE
Add -g/--debuginfo flag to wasm-emscripten-finalize

### PR DESCRIPTION
**Pull request by sbc100**
This brings this tool into parity with the existing s2wasm

### CODE REVIEW

File: src/tools/wasm-emscripten-finalize.cpp [hunk 1]
```diff
@@ -123,8 +130,8 @@ int main(int argc, const char *argv[]) {
   auto outputBinaryFlag = emitBinary ? Flags::Binary : Flags::Text;
   Output output(outfile, outputBinaryFlag, Flags::Release);
   ModuleWriter writer;
-  // writer.setDebug(options.debug);
-  writer.setDebugInfo(true);
+  writer.setDebug(options.debug);
+  writer.setDebugInfo(debugInfo);
   // writer.setDebugInfo(options.passOptions.debugInfo);
```
Comment (jgravelle-google) [inline src/tools/wasm-emscripten-finalize.cpp]: Can probably remove this line

File: src/tools/wasm-emscripten-finalize.cpp [hunk 2]
```diff
@@ -123,8 +130,8 @@ int main(int argc, const char *argv[]) {
...
   // writer.setSymbolMap(symbolMap);
```
Comment (jgravelle-google) [inline src/tools/wasm-emscripten-finalize.cpp]: Probably need to implement this line for full s2wasm parity

File: src/tools/wasm-emscripten-finalize.cpp (hunk 1 above)
Comment (sbc100) [reply src/tools/wasm-emscripten-finalize.cpp]: Done

File: src/tools/wasm-emscripten-finalize.cpp (hunk 2 above)
Comment (sbc100) [reply src/tools/wasm-emscripten-finalize.cpp]: We can do that as a followup maybe?

Comment (jgravelle-google) [reply src/tools/wasm-emscripten-finalize.cpp]: Agree, hence lgtm

**Pull request merged**